content/parts/en/datasets.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Datasets"
+html_description: "Download datasets"
+introduction: "Introductory disclaimer text presenting the varying quality depending on maintainers of the collections"
```

Comment:
```suggestion
introduction: "The datasets listed here are public [collections](https://docs.opentermsarchive.org/#collections) within the Open Terms Archive federated ecosystem. Their quality depends on their maintainers, and can vary greatly from collections curated by full-time academic research teams to on-and-off volunteers. Please make sure to read the license and maintenance commitment of each collection before publishing work based on these datasets."
```

Reply:
It was impossible to make a link in the frontmatter, so I set up a locale file in fde0973f2a2fd50f241d9f7f42a44eed2056a957